lib/perl/Genome/Model/Command/Admin/CleanupSucceeded.pm

```diff
@@ -93,21 +92,9 @@ sub _submit_abandon_jobs {
     for my $anp_id (keys %builds_by_anp) {
         my $anp = Genome::Config::AnalysisProject->get($anp_id);
 
-        my @vol = $anp->possible_volumes;
-
-        my $anp_guard = $anp->set_env;
-        my $volume_guard = Genome::Config::set_env('docker_volumes', join(' ', map { "$_:$_" } @vol));
-        my $image_guard = Genome::Config::set_env('lsb_sub_additional', sprintf('docker0(%s)', $ENV{LSB_DOCKER_IMAGE})); #use the current image regardless of the AnP config
-        unless($ENV{LSF_DOCKER_NETWORK} eq 'host') {
-            $self->fatal_message('Parent container must have LSF_DOCKER_NETWORK=host set.');
-        }
```

Comment:
what does this do exactly?

Reply:
`LSF_DOCKER_NETWORK=host` is necessary for the container to be able to communicate with child interactive LSF jobs.  This allows the terminal stdout/stderr from the subjobs to be included directly in the stdout/stderr of the current process.

(Obviously this particular line now does nothing, since this is existing code that was relocated to the helper role in this PR 😄.)